### ISSUE TITLE
Inaccurate path of config.json overriding the default config in README

### ISSUE THREAD

**Issue by hungrydonkey** [ugexe/zef#429]
In Global Configuration of README, it says that **for most cases it just means $*HOME/.zef/config.json will override an installed zef configuration file.**. But actually the path should be `$*HOME/.config/zef/config.json`

**Comment by ugexe** [comment]
Looks like that was broken in https://github.com/ugexe/zef/commit/c5635d74fe18538770842517c024577f20310929#diff-0b87bf9585ac7013f75b9ed789159ff7023c204e52596ff6670d06bbda3d9787

**Comment by hungrydonkey** [comment]
> Looks like that was broken in [c5635d7#diff-0b87bf9585ac7013f75b9ed789159ff7023c204e52596ff6670d06bbda3d9787](https://github.com/ugexe/zef/commit/c5635d74fe18538770842517c024577f20310929#diff-0b87bf9585ac7013f75b9ed789159ff7023c204e52596ff6670d06bbda3d9787)

Yeah, now `$*HOME/.zef/config.json` is not working, since in https://github.com/ugexe/zef/blob/7d6b48db23eaca06c332fa6d0ca8ecae80ae83d4/lib/Zef/Config.rakumod#L25, the `@path-candidates` doesn't contain that path.
I think README should be changed correspondingly.
